bluetooth/update/README.md

```diff
@@ -2,145 +2,133 @@
 
-After uploading data to PostgreSQL, the geometry string is needed. This can be done by adding a column using `ALTER TABLE schema.table_name ADD COLUMN geom GEOMETRY` and then filling in the column with `UPDATE TABLE schema.table_name SET geom = ST_MakePoint(lat, long)`.
+Therefore the new reader table had the following fields populated: 
 
-## Fuzzy String Matching
+`analysis_id` 		- Unique ID for each route
+`street` 			- Name of the street along the route
+`direction` 		- Route direction(East bound, West Bound, North Bound or South Bound)
+`from_street` 		- Name of the intersecting street where the route begins.
+`to_street`			- Name of the intersecting street where the route ends 
+`from_name`			- The reader name (bdit convention) at the start point of the route
+`from_id` 			- The four digit Unique bluetooth id at the start point
+`from_lat` 			- Latitude at the start point of the route
+`from_lon` 			- Longitude at the start point of the route
+`to_name` 			- The reader name (bdit convention) at the end point of the route
+`to_id`				- The four digit Unique bluetooth id at the end point
+`to_lat`			- Latitude at the end point of the route
+`to_lon`			- Longitude at the end point of the route
 
-The general process was first done to update the speed limits of roads in the centreline table. Please read the [process done for that](https://github.com/CityofToronto/bdit_data-sources/blob/master/gis/PostedSpeedLimitUpdate.md) for more detail. The gist of it is this:
+Below is the table screenshot. 
 
-* A straight line is drawn between two readers
-* Road segments with similar/exact names to the drawn line is matched to that straight line
+![new_readers_derived](img/new_table.PNG)
 
-### Explanation of Query
+## Preparatory Tables and Steps
+The following steps are utilized to create segments
+1. Get start and end geom for each analysis_id
+2. Create table with detector_id, detector_geom, centreline_int_id
+3. Join the detector's geometry to the closest centreline intersection
+4. Route the segments using street centreline's intersection `gis.centreline_both_dir` using [pgr_dijkstra].
 
-```SQL
-WITH from_join AS (
-SELECT * FROM rliu.new_bt_corridors B
-CROSS JOIN rliu.new_bt_readers A
-WHERE levenshtein(A.reader_name, B.from_intersection, 1, 1, 2) < 4)
-```
 
-Matches the corridors to the names based on similar from intersections.
+This .csv file was then imported into QGIS and pushed into the mohan schema in postgres. The geometry column is created by adding a column using `ALTER TABLE mohan.new_added_detectors ADD COLUMN from_geom GEOMETRY` we need two geometry columns: `from_geom` and `to_geom`.  Filling in the from_geom column with `UPDATE mohan.new_added_detectors SET to_geom = ST_SetSRID(ST_MakePoint(to_lon, to_lat), 4326)` and modify the query for `from_geom`.
```

Comment:
I feel like the csv can be imported directly to postgres instead of going through QGIS, since QGIS function wasn't used